package.json

```diff
@@ -15,12 +15,16 @@
     "swagger-express-mw": "^0.1.0"
   },
   "devDependencies": {
+    "eslint": "^3.18.0",
+    "npm-run-all": "^4.0.2",
     "should": "^7.1.0",
     "supertest": "^1.0.0"
   },
   "scripts": {
     "postinstall": "npm install -g multi-file-swagger",
     "start": "cd ./api/swagger/src/ && multi-file-swagger ./swagger.yaml > ../swagger.json && cd ../../../ && node app.js",
-    "test": "swagger project test"
+    "test": "npm-run-all test:swagger lint",
+    "test:swagger": "swagger_swagger_fileName=api/swagger/src/swagger.yaml swagger project test",
```

Comment:
I'm not sure myself. But the previous command just failed immediately and swagger.json isn't generated on travis. So I switched to a the yaml file which at least doesn't throw any errors.

I'll take a closer look at this tomorrow. But it doesn't break anything so let's merge this and change it in another PR.